app/src/api/learningJourney.js

```diff
@@ -62,19 +67,20 @@ export const createLearningJourneyWithJobId = async (jobId) => {
 };
 
 export const deleteLJWithLJId = async (ljId) => {
-  const deleteLJCourseRes = deleteLearningJourneyCourseWithLJId(ljId.LJId);
-  if (deleteLJCourseRes) {
-    try {
-      const deleteLJRes = await axiosLJInstance.delete(`/${ljId.LJId}`);
-      if (!deleteLJRes) {
-        throw new Error("No data returned from backend");
-      }
+  try {
+    const [deleteLJCourseRes, deleteLJRes] = await Promise.all([
+      await axiosLJCourseInstance.delete(`all/${ljId.LJId}`),
+      await axiosLJInstance.delete(`/${ljId.LJId}`),
+    ]);
+
+    if (deleteLJCourseRes && deleteLJRes) {
```

Comment:
Console logging `deleteLJCourseRes && deleteLJRes` shows that it prints the last axios response. So I reckon it would not be unlike `if (deleteLJRes)`.